tools/zbstumbler

```diff
@@ -101,7 +99,7 @@ def response_handler(stumbled, packet, channel):
         return value
 
     if args.verbose:
-        print("Received frame is not a beacon (FCF={0}).".format(pktdecode[0].encode('hex')))
+        print(("Received frame is not a beacon (FCF={0}).".format(pktdecode[0].encode('hex'))))
```

Comment:
Does this .hex work in py3, or do you need `binascii.hexlify` now?

Reply:
I believe you're correct, the hex encoding was removed in py3. recommended to convert string to bytes and use .hex()
will update